pkg/tfgen/generate.go

```diff
@@ -55,6 +56,10 @@ const (
 	maxWidth      = 120 // the ideal maximum width of the generated file.
 )
 
+// Additional files to put next schema.json when it is generated.
+var additionalSchemaFilesHook = map[string][]byte{}
```

Comment:
Eh, static state is a red flag. What would it take to get this to naturally flow through the generator invocation?